vcd/datasource_vcd_resource_list.go

```diff
@@ -685,19 +741,49 @@ func genericResourceList(resType, listMode, nameIdSeparator string, ancestors []
 		case "import":
 			identifier := ref.name
 			if ref.importId {
-				identifier = fmt.Sprintf("%s # %s/%s", ref.id, ref.parent, ref.name)
+				identifier = ref.id
 			}
 			list = append(list, fmt.Sprintf("terraform import %s.%s %s%s%s",
-				resType,
+				resourceType,
 				ref.name,
 				strings.Join(ancestors, ImportSeparator),
 				ImportSeparator,
 				identifier))
+
+			ancestorsText := ""
+			if len(ancestors) > 0 {
+				ancestorsText = strings.Join(ancestors, ImportSeparator) + ImportSeparator
+			}
+			importData.WriteString(fmt.Sprintf("# import data for %s %s%s \n", resourceType, ancestorsText, ref.name))
```

Comment:
Two more suggestions in this comment:
1. Capitalize "# import" -> "# Import", so it's easy to notice when scanning with the eyes that's it's human-oriented comment and not some auto-generate import command.
2. "data" -> "directive" (or something between those lines), as "data" does not feel entirely correct in this context.

```suggestion
			importData.WriteString(fmt.Sprintf("# Import directive for %s %s%s \n", resourceType, ancestorsText, ref.name))
```

Reply:
changed